cargo/cargo_build_script_runner/bin.rs

```diff
@@ -250,6 +250,8 @@ fn get_target_env_vars<P: AsRef<Path>>(rustc: &P) -> Result<BTreeMap<String, Str
                     .or_insert_with(Vec::new)
                     .push(value[1..(value.len() - 1)].to_owned());
             }
+        } else if ["windows", "unix"].contains(&line) {
```

Comment:
Can you explain this a bit? What exactly is this catching and how does it relate to `CARGO_CFG_(WINDOWS|UNIX)` (maybe in the form of a comment)?

Reply:
'unix' or 'windows' are one of the lines output by rustc:

```
$ rustc --print=cfg
debug_assertions
target_arch="x86_64"
target_endian="little"
target_env=""
target_family="unix"
target_feature="fxsr"
target_feature="sse"
target_feature="sse2"
target_feature="sse3"
target_feature="ssse3"
target_os="macos"
target_pointer_width="64"
target_vendor="apple"
unix
```

The resulting lines are uppercased and prefixed directly below.

Reply:
Ah! I see now, thank you! Would you be willing to add some comment like the following to help future @UebelAndre out? 😄 
```suggestion
        // `rustc --print=cfg` may include these exact values
        } else if ["windows", "unix"].contains(&line) {
```